timescaledb/quick-start/ruby.md

```diff
@@ -101,7 +101,7 @@ test:
 production:
   <<: *default
   database: my_app_db
-
+```
 
 #### Create the database
 ```
```

Comment:
```suggestion
```

Reply:
Should this be here? I noticed in the docs build we place "Now we can run the following `rake` command..." into the code snippet